graylog2-server/src/main/java/org/graylog2/grok/GrokPattern.java

```diff
@@ -17,28 +17,29 @@
 package org.graylog2.grok;
 
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.google.common.base.MoreObjects;
 import org.bson.types.ObjectId;
 
 import javax.persistence.Id;
+import java.util.Objects;
 
 @JsonAutoDetect
 public class GrokPattern {
-
     @Id
     @org.mongojack.ObjectId
     public ObjectId id;
-    
     public String name;
-    
     public String pattern;
+    public String contentPack;
 
     @Override
     public String toString() {
-        return "GrokPattern{" +
-                "id=" + id +
-                ", name='" + name + '\'' +
-                ", pattern='" + pattern + '\'' +
-                '}';
+        return MoreObjects.toStringHelper(this)
+                .add("id", id)
+                .add("name='", name)
+                .add("pattern='", pattern)
+                .add("contentPack", contentPack)
```

Comment:
Tiny thing, but we should remove those `='` from name and pattern, as `MoreObjects.toStringHelper` will take care of that